diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/LineLengthFixTest.kt

```diff
@@ -0,0 +1,35 @@
+package org.cqfn.diktat.ruleset.chapter3
+
+import org.cqfn.diktat.common.config.rules.RulesConfig
+import org.cqfn.diktat.ruleset.constants.Warnings.LONG_LINE
+import org.cqfn.diktat.ruleset.rules.LineLength
+import org.cqfn.diktat.util.FixTestBase
+import org.junit.Test
```

Comment:
```suggestion
import org.junit.jupiter.api.Test
```
Due to junit upgrade from 4 to 5. Tests with old annotation aren't included in tests suite.